### ISSUE TITLE
Search bar is not working as desired

### ISSUE THREAD

**Issue by siddharth-sable** [genicsblog/theme-files#35]
Pressing Ctrl+K should bring up the search bar, it doesn't.

Change the [Key](https://github.com/genicsblog/theme-files/blob/main/assets/js/theme.js#L68) to appropriate one, so that the search bar works as desired.

**Comment by shockz09** [comment]
![image](https://user-images.githubusercontent.com/87615572/168465307-68152272-2d0f-4389-931a-ecf8f5c4241f.png)
?

**Comment by gouravkhunger** [comment]
more context: it opens up google bar instead of genics search bar on browsers like brave, chrome and edge on windows. 

Probably changing Control / Command + K short cut to just `/` should work. Otherwise people have to disable google search bar shortcut from settings.

**Comment by AnupT16** [comment]
@gouravkhunger 
```js
document.addEventListener("keydown", function (e) {
  if (e.keyCode === 75 && (e.ctrlKey || e.metaKey)) {
    e.preventDefault();
    showSearch();
  }

  if (e.keyCode === 27) {
    hideSearch();
  }
});
```

**Comment by shockz09** [comment]
did u test it?

**Comment by AnupT16** [comment]
@shockz09 i have used it before and it works for most of browsers but if you want it support on every browser u should use jquery

**Comment by shockz09** [comment]
cool

**Comment by gouravkhunger** [comment]
Hey @AnupT16, Thanks for the idea! I'll test the `e.preventDefault()` thing on windows today (macOS and Linux didn't have such an issue that's why it went unnoticed)

**Comment by gouravkhunger** [comment]
Fixed in #36 😄 Thank you everyone!
